sql/migrations/20170507001337_roundsRewards.sql

```diff
@@ -0,0 +1,158 @@
+/*
+ * Delete table 'rounds_fees', related functions and triggers
+ * Create table 'rounds_exceptions', calculate rewards & populate it, set triggers
+ */
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS rounds_exceptions (
+	"round"          INT PRIMARY KEY,
+	"rewards_factor" INT NOT NULL,
+	"fees_factor"    INT NOT NULL,
+	"fees_bonus"     BIGINT NOT NULL
+);
+
+-- Add exception for round 27040
+INSERT INTO rounds_exceptions VALUES (27040, 2, 2, 10000000);
```

Comment:
We already discussed that before, as stated in https://github.com/LiskHQ/lisk/issues/543:
```
Apply exception only for mainnet - there are several ways to achieve that

    Clear exception table and add exception at node start (from exceptions.js)
        We decided to do it with that approach.
        Update: We cannnot do it - every rounds exception change should be performed via proper migration. Instead we check if rounds exceptions match database layer.
```
Check mechanism was added to prevent node from start if exceptions in database not match those in `exceptions.js`.